genie-client/src/main/groovy/com/netflix/genie/client/sample/GExecutionServiceSampleClient.groovy

```diff
@@ -0,0 +1,117 @@
+/*
+ *
+ *  Copyright 2015 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.genie.client.sample;
+
+import com.netflix.config.ConfigurationManager;
+import com.netflix.genie.client.ExecutionServiceClient;
+import com.netflix.genie.common.model.ClusterCriteria;
+import com.netflix.genie.common.model.FileAttachment;
+import com.netflix.genie.common.model.Job;
+import com.netflix.genie.common.model.JobStatus;
+
+import java.io.ByteArrayOutputStream;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A sample client demonstrating usage of the Execution Service Client.
+ *
+ * @author liviutudor
```

Comment:
Btw since this is on github I used my github username rather than the OneLogin one. Happy to use the Netflix one if it's better.